src/Microsoft.Data/DataFrame.IO.cs

```diff
@@ -114,7 +114,7 @@ private static Type MaxKind(Type a, Type b)
         /// <param name="guessRows">number of rows used to guess types</param>
         /// <param name="addIndexColumn">add one column with the row index</param>
         /// <returns>DataFrame</returns>
-        public static DataFrame ReadStream(Func<StreamReader> createStream,
+        public static DataFrame LoadStream(Func<StreamReader> createStream,
```

Comment:
Does this need to be `public`? I thought this was just an implementation detail of `LoadCsv`